### DISCUSSION TITLE
MVC com Blazor

### DISCUSSION THREAD

**Discussion by Igao02** [balta-io/forum#470] [Tira Dúvidas] [answered]
Balta, tudo bem??

Gostaria de uma ajuda! Eu realizei os seus cursos de Blazor, e gostaria de saber se tem como implementar o MVC com o Blazor Auto. Eu pesquisei, porém não consegui achar nenhum projeto de Blazor com o MVC aplicado. Eu gostaria de exemplos para me ajudar a separar o front do back.

Att.

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @Igao02, como vai?

Tem sim, o MVC é um padrão aberto, porém, até onde sei não existe um Framework que facilite esta implementação no Blazor (Ainda não).

Como uma alternativa de padrão no Front, eu uso e recomendo o MVVM, fizemos um evento sobre ele:
👉 https://balta.io/agenda/blazor-mvvm

**Comment by Igao02** [comment]
Balta,
Mas como ficaria por exemplo um controller no MVVM??

**Reply by andrebaltieri** [reply]
MVVM não teria controller, ele pode se conectar a um Handler por exemplo, do CQS/CQRS.
👉 https://github.com/balta-io/3001
